### ISSUE TITLE
DeprecationWarning: loaderUtils.parseQuery()

### ISSUE THREAD

**Issue by athasach** [wbuchwalter/tslint-loader#63]
I get this warning when running a build using v3.4.2:

```
DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
```

See: https://github.com/webpack/loader-utils/issues/56

**Comment by bhovhannes** [comment]
This should be fixed by #62

**Comment by blacksonic** [comment]
Fixed with version 3.4.3
